tests/NA12878_1000genomes-dragen-3.7.6/params.yaml

```
@@ -1,6 +1,5 @@
 samples:
 -
  biosample_id: NA12878 
- aln: s3://1000genomes-dragen-3.7.6/data/individuals/hg38-graph-based/NA12878/NA12878.bam
- vcf: s3://1000genomes-dragen-3.7.6/data/individuals/hg38-graph-based/NA12878/NA12878.hard-filtered.vcf.gz
-aws_no_sign_request: true
+ aln: /data/NA12878.bam
```

There is no `data` on the repo.
To revert to the s3 path

```suggestion
 aln: s3://1000genomes-dragen-3.7.6/data/individuals/hg38-graph-based/NA12878/NA12878.bam
```